src/style/BtnDesign.css

```diff
@@ -0,0 +1,12 @@
+.pos{
+    margin-left: 250px;
+}
+
+/* .btn-circle.btn-sm {
```

Comment:
If you comment something it's better to delete it and let the version control (git) save this history.